src/overview/constants.js

```diff
@@ -30,3 +30,5 @@ export const DOMAIN_TLD_PATTERN = /^(\w+\.)?[\w-]{2,}\.\w{2,3}(\.\w{2})?$/
 
 /* Pattern to match hashtags - spaces can be represented via '+' */
 export const HASH_TAG_PATTERN = /^#\w[\w+]*$/
+
+export const SHOW_TOOL_TIP = 'showTooltip'
```

Comment:
For local storage keys, we're moving all new keys to use "kebab-case", and as there's only a single level in local storage it's good to give a bit more description to these key names. Maybe `'is-overview-tooltips-shown'`?